contributing/code/stack_trace.rst

```diff
@@ -0,0 +1,182 @@
+Getting a Stack Trace
+=====================
+
+When :doc:`reporting a bug </contributing/code/bugs>` for an
+exception, it is instrumental that you provide one or several stack
+traces. To understand why, you first have to understand what a stack
+trace is, and how it can be useful to you as a developer.
+
+Anatomy of a Stack Trace
+------------------------
+
+A stack trace is called that way because it allows one to see a trail of
+function calls leading to a point in code since the beginning of the
+program. That point is not necessarily an exception. For instance, you
+can use the native PHP function ``debug_print_backtrace()`` to get such
+a trace. For each line in the trace, you get a file and a function or
+method call, and the line number for that call. This is often of great
+help for understanding the flow of your program and how it can end up in
+unexpected places, such as lines of code where exceptions are thrown.
+
+Stack Traces and Exceptions
+---------------------------
+
+In PHP, every exception comes with it own stack trace, which is
+displayed by default if the exception is not caught. When using Symfony,
+such exceptions go through a custom exception handler, which enhances
```

Comment:
```suggestion
such exceptions go through an internal exception handler, which enhances
```
maybe ad doc link to the https://symfony.com/doc/current/components/error_handler.html ?

Reply:
By custom I mean not native, and I can't see how it could be anything but internal here… as for the link, I wouldn't use it since it does not talk about the exception handler itself